httpserver/tlsManager.go

```diff
@@ -71,9 +71,7 @@ func (t *TLSCertWatcher) Start() error {
 
 	{
 		var watchErr error
-		deadlineCtx, cancel := context.WithTimeout(context.TODO(), 10*time.Second)
-		defer cancel()
-		if err := wait.PollUntilWithContext(deadlineCtx, 1*time.Second, func(ctx context.Context) (done bool, err error) {
+		if err := wait.PollUntilContextTimeout(context.TODO(), 1*time.Second, 10*time.Second, false, func(ctx context.Context) (done bool, err error) {
```

Comment:
suggestion to give meaningful names of the 1s, vs 10s input variable so when reading code, it s clear what the intention of the time specified.

Reply:
added variables for this